src/lib/receptions/sequencescapeUtils.js

```diff
@@ -80,93 +50,38 @@ const labwareForReception = async ({ requests, barcodes, requestOptions }) => {
-    // for the receptacle data, we need to map over the wells and build a request for each
-    wells_attributes: labware.relationships.receptacles.data
-      .map((receptacle) => {
-        // find the well in the included data
-        const well = findIncluded({ included, data: receptacle, type: 'wells' })
-
-        // if the well doesn't have any aliquots, we can't import it
-        if (well.relationships.aliquots.data.length === 0) {
-          return
-        }
-
-        // get the aliquot, study, sample and sample_metadata from the included data
-        const { aliquot, study, sample, sample_metadata } = getIncludedData({
-          labware: well,
-          included,
-        })
-
-        return {
-          position: well.attributes.position.name,
-          // build the request and sample objects
-          ...buildRequestAndSample({ aliquot, study, sample, sample_metadata, requestOptions }),
-        }
-        // filter out any wells that don't have aliquots
-      })
-      .filter((item) => item !== undefined),
-  }
-}
-
-/**
- *
- * @param {Object} labware Labware object from Sequencescape
- * @param {Array<Object>} included Included objects from Sequencescape
- * @param {Object} requestOptions Additional request parameters, will over-ride any
- * @returns {Object} tubes_attributes object ready for import into traction
+ * The labware will be converted from json api to nested structure labware: { receptacles: ... }
+ * will throw any errors returned from the API
+ * @param { get: Function } request Request object for the Sequencescape V2 labware API
+ * @param { String } barcodes List of comma separated barcodes to look up
+ * @param { Object } labwareRequestConfig Request parameters for retrieval of labware from Sequencescape v2 API
+ * @returns { Array<Object> } Array of normalized labware objects
  */
-const transformTube = ({ labware, included, requestOptions }) => {
-  // find the receptacle in the included data
-  const receptacle = findIncluded({
-    included,
-    data: labware.relationships.receptacles.data[0],
-    type: 'receptacles',
-  })
+const getLabware = async (request, barcodes, labwareRequestConfig) => {
+  const promise = request.get({ filter: { barcode: barcodes }, ...labwareRequestConfig })
 
-  // get the aliquot, study, sample and sample_metadata from the included data
-  const { aliquot, study, sample, sample_metadata } = getIncludedData({
-    labware: receptacle,
-    included,
-  })
+  const { success, data: { data, included = [] } = {}, errors } = await handleResponse(promise)
 
-  return {
-    barcode: labware.attributes.labware_barcode.machine_barcode,
-    // build the request and sample objects
-    ...buildRequestAndSample({ aliquot, study, sample, sample_metadata, requestOptions }),
+  if (success) {
+    return { data, included }
+  } else {
+    throw new Error(`Problem talking to Sequencescape: ${errors}`)
   }
 }
 
-const labwareTypes = {
```

Comment:
As per my previous comment I think we could put this back to remove duplication in the receptions.